### ISSUE TITLE
add input type "date"

### ISSUE THREAD

**Issue by StefanBoettcherASMB** [palasthotel/grid-wordpress#101]
on declaration:

```
			array(
				'key' => 'before',
				'label' => t('show posts before this date'),
				'type' => 'date'
			),
```

fails

**Comment by EdwardBock** [comment]
@StefanBoettcherASMB Yep, that's not a core widget yet. Should not be hard to provide.

**Comment by EdwardBock** [comment]
@StefanBoettcherASMB 

Coming with Grid 1.7.3 there is now a box widget type "input" where you can use all HTML input types.

Documentation: http://doc.the-grid.ws/dokumentation/entwickler/editor-widgets/input/
